codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/middleware/UserAgentMiddleware.kt

```diff
@@ -22,21 +22,14 @@ class UserAgentMiddleware(val settings: SwiftSettings) : MiddlewareRenderable {
         writer: SwiftWriter,
         op: OperationShape
     ) {
-        val params = middlewareParamsString(writer)
         val input = MiddlewareShapeUtils.inputSymbol(ctx.symbolProvider, ctx.model, op)
         val output = MiddlewareShapeUtils.outputSymbol(ctx.symbolProvider, ctx.model, op)
         writer.write(
-            "\$N<\$N, \$N>($params)",
+            "\$N<\$N, \$N>(serviceID: serviceName, version: \$L.version, config: config)",
```

Comment:
Instead of rendering the version as a string literal, the new static property on the service client is used.